Source/WebKit/UIProcess/API/Cocoa/WKPreferencesPrivate.h

```diff
@@ -260,6 +259,6 @@ typedef NS_ENUM(NSInteger, _WKPitchCorrectionAlgorithm) {
 @property (nonatomic, setter=_setDisplayListDrawingEnabled:) BOOL _displayListDrawingEnabled WK_API_DEPRECATED("Display list drawing is no longer supported", macos(10.12, 14.4), ios(10.0, 17.4), visionos(1.0, 1.1));
 // FIXME: We should remove this property.
 @property (nonatomic, setter=_setOfflineApplicationCacheIsEnabled:) BOOL _offlineApplicationCacheIsEnabled;
-
+@property (nonatomic, setter=_setMediaStreamEnabled:) BOOL _mediaStreamEnabled WK_API_AVAILABLE(macos(10.14)) WK_API_DEPRECATED("Media stream preference is no longer supported", macos(10.14, WK_MAC_TBA));
```

Comment:
You could alternatively say "Media stream preference is now always enabled" if that's accurate, to avoid any chance the message gets misinterpreted to mean "feature is removed."